### PR TITLE
Wire front end to point of interest API

### PR DESCRIPTION
**Pull request by micheal-w-wells**
Add point of interest interfaces, API call, and wire to trip data controls.

Front end side of https://github.com/bcgov/lucy-web/pull/956

### CODE REVIEW

File: app/src/hooks/useInvasivesApi.ts [hunk 1]
```diff
@@ -51,6 +52,18 @@ export const useInvasivesApi = () => {
...
   };
 
+  /**
+   * Fetch points of interest by search criteria.
+   *
+   * @param {pointsOfInterestSearchCriteria} activitiesSearchCriteria
+   * @return {*}  {Promise<any>}
+   */
+  const getPointsOfInterest = async (pointsOfInterestSearchCriteria: IPointOfInterestSearchCriteria): Promise<any> => {
+    const { data } = await api.post(`/api/points-of-interest/`, pointsOfInterestSearchCriteria);
```
Comment (NickPhura) [inline app/src/hooks/useInvasivesApi.ts:62]: Just notice that the api route doesnt quite match the filename in the api.  I think its just the 's' on 'points'

Comment (micheal-w-wells) [reply app/src/hooks/useInvasivesApi.ts:62]: good catch thanks

Comment (NickPhura) [reply app/src/hooks/useInvasivesApi.ts:62]: Does this still need updating?

File: app/src/interfaces/useInvasivesApi-interfaces.ts [hunk 2]
```diff
@@ -60,7 +61,7 @@ export interface IActivitySearchCriteria {
 }
 
 /**
- * Create activity endpoint post body.
+ * Create point_of_interest endpoint post body.
```
Comment (NickPhura) [inline app/src/interfaces/useInvasivesApi-interfaces.ts]: Should still be activity

**Review by NickPhura — DISMISSED** [review]
Just the 2 minor comments

**Review by NickPhura — APPROVED** [review]

**Pull request merged**